src/govuk/components/tabs/tabs.mjs

```diff
@@ -230,19 +230,17 @@ Tabs.prototype.unsetAttributes = function ($tab) {
  * Handle tab link clicks
  *
  * @param {MouseEvent} event - Mouse click event
- * @returns {void | false} Returns void, or false within tab link
+ * @returns {void} Returns void
  */
 Tabs.prototype.onTabClick = function (event) {
-  if (!event.target.classList.contains('govuk-tabs__tab')) {
```

Comment:
We've removed as `event.currentTarget` will be the tab link (not a child element)

We know the IE8 polyfill adds `event.currentTarget` support but just need to verify if it's clever enough to handle **target** vs **currentTarget** (might just alias them as the same thing)

Reply:
Yep from our [polyfill](https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/vendor/polyfills/Event.mjs#L140-L142)

```js
event.currentTarget = element;
event.relatedTarget = event.fromElement || null;
event.target = event.target || event.srcElement || element;
```

Where **element** is `this` (current target)